packages/web3-shared/base/src/specs/index.ts

```diff
@@ -386,18 +386,24 @@ export interface NonFungibleAsset<ChainId, SchemaType> extends NonFungibleToken<
  * Authorization about a fungible token.
  */
 export interface FungibleTokenAuthorization<ChainId, SchemaType> {
-    amount: string
-    recipient: string
-    token: FungibleToken<ChainId, SchemaType>
+    tokenInfo:  Pick<FungibleToken<ChainId, SchemaType>, 'address' | 'logoURL' | 'symbol' | 'name'>
+    address: string
+    name: string | undefined
+    logo: React.ReactNode | undefined
```

Comment:
```suggestion
    spender: {
        address: string
        name: string | undefined
        logo: React.ReactNode | undefined
    }
```

Reply:
Rename `FungibleTokenAuthorization` to `FungibleTokenSpenderAuthorization`, and added code comments.